p9/file.go

```diff
@@ -132,6 +132,15 @@ type File interface {
 	// On the server, FSync has a read concurrency guarantee.
 	FSync() error
 
+	// Lock locks the file. The operation as define in 9P2000.L
+	// is fairly ambitious, and in most implementations it is
+	// just using flock.  But we provide the full API for anyone
+	// who thinks they want it.
+	//
+	// Many implementations do nothing, successfully;
+	// or call flock (lock the whole file).
+	Lock(pid, locktype, flags int, start, length uint64, client string) error
```

Comment:
Can you document the args the same way you did for the msg here

Reply:
ptal.
thanks!